datasets/woz/wozdeelobjecten/v1.1.0.json

```diff
@@ -0,0 +1,131 @@
+        "description": "Uniek volgnummer van de toestand van het object."
+      },
+      "registratiedatum": {
+        "type": "string",
+        "format": "date-time",
+        "description": "De datum waarop de toestand is geregistreerd."
+      },
+      "wozdeelobjectnummer": {
+        "type": "string",
+        "$comment": "should become: wozdeelobjectnummer, not possible atm.",
+        "description": "De unieke identificatie van een WOZ-deelobject"
+      },
+      "deelnummer": {
+        "type": "string",
+        "description": "Unieke identificatie WOZ-deelobject binnen het WOZ-object"
+      },
+      "wozobjectnummer": {
+        "type": "string",
+        "description": "Van welk WOZ-object is dit WOZ-deelobject een onderdeel"
+      },
+      "soortobjectCode": {
+        "type": "integer",
+        "provenance": "$.soortobject.code",
+        "description": "Nadere aanduiding van het feitelijk gebruik  (SOC) van een WOZ-object zoals dat ten grondslag heeft gelegen aan de waardebepaling en voor zover relevant voor de afnemers, uitgedrukt in een code code"
+      },
+      "soortobjectOmschrijving": {
+        "type": "string",
+        "provenance": "$.soortobject.omschrijving",
+        "description": "Nadere aanduiding van het feitelijk gebruik  (SOC) van een WOZ-object zoals dat ten grondslag heeft gelegen aan de waardebepaling en voor zover relevant voor de afnemers, uitgedrukt in een code omschrijving"
+      },
+      "beginGeldigheid": {
+        "type": "string",
+        "format": "date",
+        "description": "Een aanduiding op welk tijdstip het object is ontstaan"
+      },
+      "eindGeldigheid": {
+        "type": "string",
+        "format": "date",
+        "description": "Een aanduiding op welk tijdstip het object is beëindigd"
+      },
+      "isVerbondenMetVerblijfsobject": {
+        "type": "object",
+        "properties": {
+          "identificatie": {
+            "type": "string"
+          },
+          "volgnummer": {
+            "type": "integer"
+          }
+        },
+        "relation": "bag:verblijfsobjecten",
+        "description": "Aan welk verblijfsobject heeft het WOZ-deelobject een relatie"
+      },
+      "isVerbondenMetLigplaats": {
+        "type": "object",
+        "properties": {
+          "identificatie": {
+            "type": "string"
+          },
+          "volgnummer": {
+            "type": "integer"
+          }
+        },
+        "relation": "bag:ligplaatsen",
+        "description": "Aan welk ligplaats heeft het WOZ-deelobject een relatie"
+      },
+      "isVerbondenMetStandplaats": {
+        "type": "object",
+        "properties": {
+          "identificatie": {
+            "type": "string"
+          },
+          "volgnummer": {
+            "type": "integer"
+          }
+        },
+        "relation": "bag:standplaatsen",
+        "description": "Aan welk standplaats heeft het WOZ-deelobject een relatie"
+      },
+      "heeftPand": {
```

Comment:
Ik zou hier `heeftPanden` verwachten. Maar, wellicht staat het op deze manier in stelselpedia?

Reply:
Niet per se zo te [zien](https://www.amsterdam.nl/stelselpedia/woz-index/catalogus-woz/objectklasse-woz-object/) en binnen GOB heet dit veld `heeft_pand` dus ik ga er vanuit dat het in AMS echt `heeftPand` moet zijn.

Reply:
PR staat _for now_ on hold. Er moet eerst verder worden uitgezocht wat er nu mis gaat.
Bedankt voor de trigger Jan.